Section 2 - Read, Generate, Modify Congiruations/attributes.tf

```diff
@@ -14,6 +14,46 @@ output "eip" {
 
 resource "aws_s3_bucket" "mys3" {
```

Comment:
**Tenable.cs remediation**
 
**[HIGH]: Ensure at-rest server side encryption (SSE) is enabled using default encryption keys for AWS S3 Buckets**
 
**Vulnerability**
AWS S3 Buckets do not have at-rest server side encryption enabled to protect sensitive data.
 
**Remediation**
Server-side encryption protects data at rest. Amazon S3 encrypts each object with a unique key. As an additional safeguard, it encrypts the key itself with a key that it rotates regularly. Amazon S3 server-side encryption uses one of the strongest block ciphers available to encrypt your data, 256-bit Advanced Encryption Standard (AES-256).
 In Terraform - 
 Add the 'server_side_encryption_configuration' block to ensure server side encryption is enabled. Set 'sse_alogrithm' set to 'AES256'.

source: tenablecs-